js/pages/pathways/components/manager.html

```diff
@@ -45,7 +45,15 @@
 				key: 'utilities',
 				componentName: 'pathway-utils',
 				componentParams: $component.componentParams,
-			}
+			},
+            {
+                title: 'Messages', 
+                key: 'messages',
+                componentName: 'warnings',
+                componentParams: $component.componentParams,
```

Comment:
Pls note each tab has it's own `componentParams`, therefore we don't obliged to translate the whole set of parameters to every tab

Reply:
fixed